src/ansys/dpf/post/static_mechanical_simulation.py

```diff
@@ -4148,6 +4148,873 @@ def creep_strain_equivalent_elemental(
             skin=skin,
         )
 
+    def thermal_strain(
```

Comment:
@cbellot000 I do not see why it would not. I'll implement those too. I wanted to treat the issue quickly so I made this PR for Static, but I can definitely change its scope.

Reply:
@cbellot000 I checked and Mechanical does not propose thermal strain results for modal or harmonic analyses so I'd rather not expose that unless someone specifically asks or if you are aware that this is a use-case.